### ISSUE TITLE
DaedalusX64 crashes at start with colorful static on both screens

### ISSUE THREAD

**Issue by Agetian** [masterfeizz/DaedalusX64-3DS#98]
Trying to run DaedalusX64 v1.0 on a New 2DS XL running Luma3DS v11.0 CFW.
Installed fresh from Github releases (tried both the 3dsx version and the cia version), unpacked 3ds.zip into the root of the SD card as instructed. Trying to start the emulator results in a colorful static screen as demonstrated below:
![изображение](https://user-images.githubusercontent.com/24357441/196449418-0ed11fc3-4edf-4ff6-b9c9-2d186f397846.png)

The only difference between the 3dsx and the cia version is that the static pattern is slightly different, otherwise the behavior is the same.
If I try pressing the buttons (typically Start and Select), eventually it'll lead to an ARM11 crash as shown below (this crash seems to happen on the cia version, while the 3dsx version instead throws me out back into the homebrew launcher menu):
![изображение](https://user-images.githubusercontent.com/24357441/196449757-ea4499b3-c1e4-4c01-bf1a-60a2a8ab9f30.png)

Tried reinstalling fresh, including wiping and unpacking 3ds.zip again, making sure the files are in the right location (3ds/DaedalusX64 on the SD card), but to no avail.

The CFW itself was installed a few years back (and later upgraded to 11.0) and works correctly, every other homebrew app I tried works fine (whether 3dsx or cia), so I don't expect it to be a hardware/CFW issue with my 2DS XL. I did dump my DSP when I first installed the CFW and I checked that I still have the dspfirm.cdc file under the "3ds" folder.

**Comment by Trickygamers** [comment]
I have the same issue on my new 2ds xl glad to know my 2ds isn't faulty.

**Comment by WaterMelon4938** [comment]
Are there ROMs in the rom folder?

**Comment by Agetian** [comment]
Yes, that does the trick, thanks! Turns out the Roms folder doesn't exist in the 3ds.zip archive in the first place (even an empty one), and without it and at least one ROM, DaedalusX64 crashes at start. It's certainly not very useful to launch the emulator without any ROMs, but the behavior in case there are none (and no Roms folder) is still somewhat confusing and puzzling, I would recommend, if possible, to either alter this behavior to show an error message, or, if it's not possible, to document this behavior explicitly in the release notes / README file.

**Comment by Invictaz** [comment]
@masterfeizz confirmed on Twitter that he forgot to add the no roms found screen leading to this behavior.

Solution: make Roms folder and put at least one rom in it in the correct format.

Static will still display but only for 1 second.

**Comment by L0N3W0LFJR** [comment]
I tried this and it still crashes can anyone help me please

**Comment by atchrex** [comment]
I had the same problem
I think you downloaded it from Universal Updater and it's an error

install it from the official Github and rename your folder to "roms" and not to Roms
the Github link: https://github.com/DaedalusX64/daedalus/releases/download/1.1.8/DaedalusX64_1_1_8.zip

**Comment by DekuScrubs** [comment]
> I had the same problem I think you downloaded it from Universal Updater and it's an error
> 
> install it from the official Github and rename your folder to "roms" and not to Roms the Github link: https://github.com/DaedalusX64/daedalus/releases/download/1.1.8/DaedalusX64_1_1_8.zip

Still getting the same static screen regardless of file source, tried adding a Roms folder and a roms folder in case capitalization matters, it didnt

**Comment by RavenPlayz1** [comment]
> > I had the same problem I think you downloaded it from Universal Updater and it's an error
> > install it from the official Github and rename your folder to "roms" and not to Roms the Github link: https://github.com/DaedalusX64/daedalus/releases/download/1.1.8/DaedalusX64_1_1_8.zip
> 
> Still getting the same static screen regardless of file source, tried adding a Roms folder and a roms folder in case capitalization matters, it didnt

Any thing new to try and fix it?

**Comment by RavenPlayz1** [comment]
> > I had the same problem I think you downloaded it from Universal Updater and it's an error
> > install it from the official Github and rename your folder to "roms" and not to Roms the Github link: https://github.com/DaedalusX64/daedalus/releases/download/1.1.8/DaedalusX64_1_1_8.zip
> 
> Still getting the same static screen regardless of file source, tried adding a Roms folder and a roms folder in case capitalization matters, it didnt

Anything new to try and fix it? Because I'm having the same issue. Nothing is fixing it.

**Comment by SusyVEXuser** [comment]
sus

**Comment by SusyVEXuser** [comment]
idk im having the same static issue too

**Comment by halian** [comment]
Occurs for me on N3DSXL with the `.cia` version, no matter the capitalization of the `roms` folder.

**Comment by SenseiUsagi** [comment]
I managed to fix it on my end, I have a "ROM" folder and a "roms" folder, idk which one matters but both have actual .n64 roms and now it works
